hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/extension/OperationSubsystemStart.java

```diff
@@ -44,7 +44,17 @@ public void execute(OperationContext opContext, ModelNode model) throws Operatio
                 LOGGER.debug("Asked to start the Hawkular Monitor service");
             }
             if (!service.isMonitorServiceStarted()) {
-                service.startMonitorService();
+                Thread newThread = new Thread(new Runnable() {
+                    @Override
+                    public void run() {
+                        try {
+                            service.startMonitorService();
+                        } catch (Throwable t) {
+                        }

```

Comment:
This reason why I do not log anything here is because startMonitorService() will already log that error - any Throwable thrown from method startMonitorService is first logged by that method, see here:

https://github.com/jmazzitelli/hawkular-agent/blob/fb0e9878953ffc2bbd2922277af7f154e6568176/hawkular-wildfly-agent/src/main/java/org/hawkular/agent/monitor/service/MonitorService.java#L646

You can see startMonitorService() has a large try-catch that surrounds the entire start up code and the catch block logs the exception. Thus to avoid simply duplicating that error log message, I didn't log that exception a second time here.